app/controllers/resend_renewal_email_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class ResendRenewalEmailController < ApplicationController
+  def new
+    authorize
+
+    begin
+      RenewalReminderMailer.first_reminder_email(registration).deliver_now
+
+      flash[:message] = I18n.t("resend_renewal_email.messages.success", email: registration.contact_email)
```

Comment:
With this merged its going to make implementing [RUBY-467 Add option to resend confirmation email](https://eaflood.atlassian.net/browse/RUBY-467) much easier as well 🎉 😁 

So on the basis we know this other story is needed, and this is a pattern we'll want to use in WCR, is there a way we can move the implementation for assigning `flash[:message]`, and `flash[:error]` somewhere else? A controller concern for example?

Then rather than duplicating this across controllers, we can just call something like

```ruby
flash_success("Email sent. Well done you!")
flash_error("You broke it", "You appear to have tried using the system and broke it. Someone's in trouble!")

# ...

def flash_success(message)
  flash[:message]
end

def flash_error(message, description)
  flash[:error] = error
  flash[:error_details] = description
end
```

It is pre-emptive refactoring, but I'm 99.999% sure this will be reused!

Reply:
Sure! Will do that :)